src/session_impl.cpp

```diff
@@ -1897,7 +1891,8 @@ namespace aux {
 			// only try to open the IPv6 port if IPv6 is installed
 			if (supports_ipv6())
 			{
-				s = setup_listener("::1", false, m_listen_interface.port()
```

Comment:
the main problem here was the "::1" typo. which clearly should be "::". there's another one in the next hunk down